src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -1,12 +1,116 @@
-import Box from '@mui/material/Box'
+import { MenuItem, TextField, Typography, Box } from '@mui/material'
 
 import { styles } from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep.styles'
+import translations from '~/constants/translations/en/become-tutor.json'
+import img from '~/assets/img/tutor-home-page/become-tutor/general-info.svg'
+import { useState } from 'react'
 
 const GeneralInfoStep = ({ btnsBox }) => {
+  const [countryList, setCountryList] = useState([])
+  const [cityList, setCityList] = useState([])
+  const [selectedCountry, setSelectedCountry] = useState('')
+  const [selectedCity, setSelectedCity] = useState('')
+  const [professionalStatus, setProfessionalStatus] = useState('')
+  const [charCount, setCharCount] = useState(0)
+
+  const countryCityMap = {
```

Comment:
I think there is a backend task to fetch countries and respective cities #94 
So I believe this logic is redundant